packages/matter.js/src/cluster/server/GroupsServer.ts

```diff
@@ -71,7 +71,7 @@ export class GroupsManager {
     }
 }
 
-export const GroupsClusterHandler: () => ClusterServerHandlers<typeof GroupsCluster> = () => {
+ClusterServerFactory.registerClusterDefaults(GroupsCluster, () => {
```

Comment:
Maybe we could here already set the default attribute initial values that enhance the defined defaults - see the name support thingy.

Reply:
Yeah, that seems right to me.  I added the defaultInitialAttributeValues override to the registration function because you asked for that but I wasn't sure about making it a partial because you'd lose some of the type safety.  But if we check manually farther down then that would mitigate that issue.

Reply:
And as said above: Even with the risk of "duplicate code" (aka defaults also listed in attribute initial values) I have no issue with this because then you see all attributes used in one place because sometimes they depend on each other